### ISSUE TITLE
TAG Feedback: high-performance signal for iframes

### ISSUE THREAD

**Issue by travisleithead** [immersive-web/webxr#346]
Following up on a few extra comments in TAG review w3ctag/design-reviews#185

When the TAG discussed WebXR feedback at TPAC 2017, one of the issues that came up was a desire for a 'high performance' (or similar) declaration for iframes, as a way to signal that the iframe's purpose will be to run a render-loop for XR. An implementation could use such a signal to de-prioritize activity outside of the iframe when it is processing XR content, for example.

(Note also the potential for abuse, if say, Ad frameworks started marking their Ad iframes with this signal--so care should be taken in its design.)

**Comment by cwilso** [comment]
This seems like a system-wide web platform feature request with a LOT of concerning implications, not a WebXR feature?

**Comment by toji** [comment]
To expand on what Chris said, this is a subject that we loop back on semi-frequently within the working group since the feedback was originally given, and the conclusion is always that it does feel like something that sits outside the scope of the WebXR API for a couple of reasons:

 1. This functionality, if implemented, would benefit many types of media, not just VR (though admittedly our API is more performance-sensitive than most.) It would be awkward at best to require that the WebXR API be implemented on devices with no potential for immersive computing simply because there was a useful performance hint that was tied to our spec.
 1. The side effects of such a flag seem like they would run deep in the browser, potentially impacting a great many other features that WebXR otherwise has no direct association with. Our group is not in a position to handle those sorts of potentially-wide-ranging ramifications throughout the web platform. Especially given the aforementioned potential for abuse.

In short, it doesn't feel like our specification is the appropriate place for such a feature, not does our group necessarily contain the appropriate domain expertise to define it successfully. However, if such a feature _is_ introduced to the web platform, we will happily advocate it's use with immersive content!
